superset-frontend/src/explore/components/controls/DndColumnSelectControl/DndFilterSelect.tsx

```diff
@@ -304,9 +311,14 @@ export const DndFilterSelect = (props: DndFilterSelectProps) => {
         sqlExpression: (droppedItem as AdhocMetric)?.translateToSql(),
       });
     }
-    return new AdhocFilter({
+    const config: Partial<AdhocFilter> = {
       subject: (droppedItem as ColumnMeta)?.column_name,
-    });
+    };
+    if (isFeatureEnabled(FeatureFlag.UX_BETA)) {
+      config.operator = OPERATOR_ENUM_TO_OPERATOR_TYPE[Operators.IN].operation;
+      config.operatorId = Operators.IN;
+    }
```

Comment:
When we drop a non-string column, the UX feels slightly unintuitive (here I dropped the `num` column):

![image](https://user-images.githubusercontent.com/33317356/127155622-e5226403-35d0-4be5-bf06-242dbb619d4e.png)

Since we have the full set of columns along with their `type_generic`, maybe we could consider having different behavior for numeric and temporal column types while we're at it?

Reply:
good idea! it would be great to have the numeric slider in adhoc filter eventually when we redesign the entire component. 
For temporal columns, i doubt users would use adhoc filter for it. 
For numeric, we can use "=" or "<"? either way, the preselection is not gonna trigger any query, or adding additional clicks to selection, so the change is harmless. it really depends on what our best guess is without further data.